docs/zkapps/tutorials/02-private-inputs-hash-functions.mdx

```diff
@@ -24,99 +24,194 @@ zkApp programmability is not yet available on the Mina Mainnet. You can get star
 
 :::
 
-:::note
 
-This tutorial was last tested with [SnarkyJS](https://www.npmjs.com/package/snarkyjs) 0.8.0.
+# Tutorial 2: Private Inputs and Hash Functions
 
-:::
+In the [Hello World](hello-world) tutorial, you built a basic zkApp smart contract in SnarkyJS with a single state variable that could be updated if you knew the square of that number.
```

Comment:
nit: 
built a basic zkApp smart contract `in` SnarkyJS -> built a basic zkApp smart contract `with` SnarkyJS